diff --git a/node_modules/react-native/React/Base/RCTUtils.m b/node_modules/react-native/React/Base/RCTUtils.m
index 4f14114..f133d30 100644
--- a/node_modules/react-native/React/Base/RCTUtils.m
+++ b/node_modules/react-native/React/Base/RCTUtils.m
@@ -825,9 +825,9 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
     const char *fileSystemCString = [imageURL fileSystemRepresentation];
     if (fileSystemCString != NULL) {
       NSString *filePath = [NSString stringWithUTF8String:fileSystemCString];
-      if (filePath.pathExtension.length == 0) {
+      /* if (filePath.pathExtension.length == 0) {
         filePath = [filePath stringByAppendingPathExtension:@"png"];
-      }
+      } */
       image = [UIImage imageWithContentsOfFile:filePath];
     }
   }
